packages/resolver/lib/sources/truffle.ts

```diff
@@ -0,0 +1,37 @@
+import path from "path";
+import fse from "fs-extra";
+
+import { ResolverSource } from "../source";
+
+export class Truffle implements ResolverSource {
+  async resolve(importPath: string) {
+    let body, filePath;
+    if (importPath === "truffle/Console.sol") {
+      const actualImportPath =
+        // @ts-ignore
+        typeof BUNDLE_VERSION !== "undefined"
+          ? path.resolve(__dirname, path.basename(importPath))
+          : path.resolve(
+              __dirname,
+              "../../../../core/lib/logging",
```

Comment:
I don't like how this is reaching into @truffle/core... especially considering that we might want to do the same here with Assert+DeployedAddresses. I think those contracts need a new home. Any ideas? I was thinking we could have @truffle/solidity-logging for Console

Reply:
We could, it just feels gross to have them all there. Maybe the new package could be `@truffle/solidity-libs`, for all three?

Reply:
It seems to me that all these would be at home in `resolver`. They are basically libraries that the `truffle` source resolves. That feels pretty appropriate to me. What do you think?

Reply:
The decision was made to move all these templates into the `resolver` package.